charts/tidb-cluster/templates/scripts/_start_tikv.sh.tpl

```diff
@@ -39,6 +39,11 @@ ARGS="--pd={{ template "cluster.scheme" . }}://${CLUSTER_NAME}-pd:2379 \
 --config=/etc/tikv/tikv.toml
 "
 
+if [ ! -z "${STORE_LABELS:-}" ]; then
```

Comment:
can we add this script snippets to `.Values.tikv.preStartScript` to avoid modify the `_start_tikv.sh.tpl`?

Reply:
It's ok to me. I would change to use snippets soon.

Reply:
updated